README.md

```diff
@@ -16,14 +16,16 @@ If this is the first time you're working with this repository, install the depen
 
 ### Exporting the sources to `mozilla-central`
 
-1. Run `npm run jake export-mc` for Desktop or `npm run jake export-mc-android` for Android.
-2. Find the exported files in your `mozilla-central` directory, ready to commit.
+1. Ensure the version number is bumped in `package.json` and `src/manifest.json`, appropriately (see [Versioning Scheme](https://github.com/mozilla/webcompat-addon/wiki/Versioning-Scheme) for more info).
+2. Run `npm run jake export-mc` for Desktop or `npm run jake export-mc-android` for Android.
+3. Find the exported files in your `mozilla-central` directory, ready to commit.
 
 ### Exporting the sources into Android Components
 
-1. Make sure the `EXPORT_AC_LOCATION` environment variable is set to the root of your Android Components checkout.
-2. Run `npm run jake export-ac`.
-3. Find the exported files in your Android Components directory, ready to commit.
+1. Ensure the version number is bumped in `package.json` and `src/manifest.json`, appropriately (see [Versioning Scheme](https://github.com/mozilla/webcompat-addon/wiki/Versioning-Scheme) for more info).
+2. Make sure the `EXPORT_AC_LOCATION` environment variable is set to the root of your Android Components checkout.
```

Comment:
On a second thought, if you're noting the ENV var here, let's add it to the mc-exporter as well.